spec/requests/admin/application_hearing_results_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+RSpec.describe "POST /admin/court_application/:id/result/:hearing_id/:publish_to" do
+  let(:hearing) { create(:hearing) }
+  let(:court_application) { create(:court_application) }
+  let(:publish_to) { "dev" }
+  let(:headers) { { 'Authorization': authorisation } }
+  let(:authorisation) { ActionController::HttpAuthentication::Basic.encode_credentials(ENV["ADMIN_USERNAME"], ENV["ADMIN_PASSWORD"]) }
+
+  before do
+    allow(HearingResulter).to receive(:call).with(hearing_id: hearing.id, publish_to:).and_return(outcome)
+    post(admin_court_application_result_hearing_path(court_application, hearing, publish_to), headers:)
```

Comment:
since it is a request spec it would be nicer to tests the actual URL structure not just rails the routing system